core/store/migrations/migrate_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"testing"
 	"time"
 
+	"gopkg.in/guregu/null.v3"
```

Comment:
Can we use null.v4 for new things>

Reply:
ah actually the JobSpec struct uses v3 still, so I have to use v3 for this test